src/Article.jsx

```diff
@@ -0,0 +1,17 @@
+import "./All.css";
+
+export default function Article({ article }) {
+  return (
+    <div key={article.id} className="article">
+      <button className="article-title">
+        Title of the article: {article.title}
+      </button>
```

Comment:
If it isn't going to do anything when you click on it then it shouldn't be a `<button>` tag, this will be a problem for screen readers 

If however it's part of some master game plan you have to make it clickable then feel free to ignore this...